test/commands/distribute/release/mockWorker.js

```diff
@@ -0,0 +1,6 @@
+const { parentPort } = require("worker_threads");
```

Comment:
Do we still need it?

Reply:
The whole file will be removed in tests update.
@annakocheshkova and @Jamminroot is working on it.